Formalisation(shacl)/Core/PiecesShape/DataService.ttl

```diff
@@ -0,0 +1,36 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix dash: <http://datashapes.org/dash#> .
+
+
+:DataServiceShape a sh:NodeShape ;
```

Comment:
Tests fail on a missing prefix declaration for `:`.

Reply:
feat: add Prefix "%s:" in DataService.ttl